### ISSUE TITLE
Window icon and title do not work on Ubuntu 14.04 Unity or GNOME 3

### ISSUE THREAD

**Issue by elldritch** [electron/electron#463]
Setting the `icon` or `title` options when I construct a new `BrowserWindow` don't do anything; when I run the application, the title still shows up as 'Atom' and the icon is still the Atom text editor icon.

i.e. this code:

``` javascript
main_window = new BrowserWindow({
  width: 800,
  height: 600,
  icon: __dirname + '/assets/img/some_icon.jpg',
  title: 'some title here'
});
```

still results in this window title/icon:
![screenshot from 2014-07-06 14 06 59](https://cloud.githubusercontent.com/assets/2092654/3489966/d6a70e5a-0551-11e4-93d2-86e91569133e.png)

**Comment by zweicoder** [comment]
![image](https://cloud.githubusercontent.com/assets/3684550/12107201/97ca4084-b3a2-11e5-9392-6c05b689bdad.png)
Having made a `.desktop` file for Ubuntu 14.04 this is what happens. Is this the intended behavior?

**Comment by LucasIsasmendi** [comment]
My solution was create a desktopIcon64.png, 64px image size
Then in the main.js
BrowserWindow({icon: __dirname+'/pathtoimage/desktopIcon64.png'})
I am in ubuntu 14.04, electron v1.2.3

**Comment by samuelstein** [comment]
for me i could fix the problem on ubuntu 16.04 like that: `setTimeout(() => mainWindow.webContents.openDevTools({mode: 'detach'}), 1000)`

the problem was the fast startup of the devtools.
